contributed/matching_fof_mlz_blending.ipynb

```diff
@@ -0,0 +1,805 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Make MLZ input catalog of [1-1], [2-1], and [1-2] groups from fof matching.\n",
+    "Editted by HyeYun Park, adding MLZ catalog making to 'matching_fof.ipynb' <br>\n",
```

Comment:
Also, for goals in the beginning (or somewhere else, but in one place), it would be nice to explicitly say how FoF matching relates to the generation of 4 sets of training/test catalogs -- that you're getting possible 1-1, 1-2, 2-1 matches between truth and coadd and generating 4 sets that differ in which way.